src/features/VisualNovel/pages/OverviewPage/OverviewPage.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { FC, Fragment, useEffect, useState } from 'react';
+import { Heading, Link } from '@chakra-ui/react';
+
+import languageCodes from 'iso-639-1';
+import { useParams } from 'react-router';
+import cls from './OverviewPage.module.css';
+import { StaffRoles, STAFF_ROLES } from '../../../../utils/types/staffRoles';
+import { VisualNovelLinks } from '../../../../utils/types/visualNovelLinks';
+import { TagBlock } from '../../components/TagBlock/TagBlock';
+import { CharacterCard } from '../../components/CharacterCard/CharacterCard';
+import { useVisualNovelQuery, useCharactersQuery, useReleasesQuery, useTagsQuery } from '../../queries';
+import { Release } from '../../../../models/release';
+
+/**
+ * Overview tab page.
+ */
+export const OverviewPage: FC = () => {
+  const { id } = useParams();
+  const [developers, setDevelopers] = useState<string[]>([]);
+  const [publishers, setPublishers] = useState<Record<string, string[]>>(
+    languageCodes.getAllCodes().reduce((acc, val) => ({ ...acc, [val]: [] as string[] }), {}),
+  );
+
+  const { isLoading, error, data: visualNovel } = useVisualNovelQuery(id);
+  const {
+    isLoading: isReleasesLoading,
+    error: releasesError,
+    data: releases,
+  } = useReleasesQuery(id);
+
+  /**
+   * Fills developers and publishers arrays.
+   * @param releasesData Releases.
+   */
+  const fillDeveloperTeam = (releasesData: Release[]): void => {
+    setDevelopers(Array.from(new Set(releasesData
+      .map(release => release.producers
+        .filter(p => p.isDeveloper)
+        .map(p => p.name))
+      .flat())));
+
+    /**
+     * Filling publisher object with unique values.
+     */
+    const publisherReleases = releasesData.filter(release => release.producers.filter(p => p.isPublisher)).flat();
+    const publishersCopy = { ...publishers };
+    publisherReleases.forEach(release => {
+      release.languages.forEach(lang => {
+        const publisherNames = release.producers.map(p => p.name);
+        const uniquePublisherNames = Array.from(new Set(publishersCopy[lang].concat(publisherNames)));
+        publishersCopy[lang] = uniquePublisherNames;
+      });
+      setPublishers(publishersCopy);
+    });
+  };
```

Comment:
So `publishers` state is updated per each release? How does it work? Maybe just create result object after all iterations and call `setPublishers` one time?